src/app/hero-list/hero-list.component.ts

```diff
@@ -16,6 +16,11 @@ export class HeroListComponent implements OnInit {
   prevPage: string;
   loading: boolean = false;
 
+  extractId(url: string){
+    const urlArr = url.split('/');
+    return parseInt(urlArr[urlArr.length-2]);
+  }
+
```

Comment:
In case you're wondering. Hero object coming from API does not contain it's id as a value so I have to extract it from it's url.